app/views/fragments/promotion/pricingCta.scala.html

```diff
@@ -0,0 +1,15 @@
+@import com.gu.subscriptions.DigipackCatalog
+@import com.gu.memsub.promo.Promotion.AnyPromotion
+@import com.gu.memsub.promo.PromoCode
+@import model.DigitalEdition
+
+@(edition: DigitalEdition, catalog: DigipackCatalog, promoCode: PromoCode, promotion: AnyPromotion)
+<div class="pricing-cta__pricing">
```

Comment:
It's particularly confusing because this is inside a template - looking at this template I would start wondering if I was supposed to wrap the template in `<div class=pricing-cta></div>` wherever it was used. But looking at your usages of it, it seems this isn't the case.

Generally you would want your BEM components to match up with your templates, so you can use the templates in a self-contained way and not couple them to the environment they would be used in